docs/introduction.rst

```diff
@@ -15,29 +15,29 @@
 
-It provides a scalable, secure, robust infrastructure for collecting data from sensors, processing data at the edge using intelligent data pipelines and transporting data to historian and other management systems. Fledge also allows for edge based event detection and notification and control flows as a result of events, stimulus from upstream systems or user action. Fledge can operate over the unreliable, intermittent and low bandwidth connections often found in industrial or rugged environments.
+Fledge provides a scalable, secure, robust infrastructure for collecting data from sensors, processing data at the edge using intelligent data pipelines and transporting data to historian and other management systems. Fledge also allows for edge based event detection and notification and control flows as a result of events, stimulus from upstream systems or user action. Fledge can operate over the unreliable, intermittent and low bandwidth connections often found in industrial or rugged environments.
 
 Typical Use Cases
 -----------------
 
-Fledge may be used in a number of different ways and for different reasons to suit your operational data requirements, listed here are just a few of the ways Fledge is being used.
+The depth and breadth of Industrial IoT use cases is considerable.   Fledge is designed to address them.  Below are some examples of typical Fledge deployments.
 
 Unified data collection
-    There are many situation that have arisen over time whereby a heterogeneous control system allowed data to collected and processed in a single system. With the advent of the internet of things and a growing number of after market sensors becoming available this is no longer the case and many installations find themselves in a situation whereby they have silos of data that reside in different systems. Fledge is designed to help you illuminate those silos by providing a very flexible data collection and distribution mechanism.
+    The industrial edge is one of the more challenging in computing.  Today there are over 100 different protocols, no standards in machine data definitions, different types of data (time-series, vibration, array, image, radiometric, transactional, etc.), sensors producing bytes/hr to gigs/hr all in environments with network, power and environmental challenges.   This diversity creates pain in managing, scaling, securing and orchestrating industrial data.  Ultimately resulting in silos of data with competing context.   Fledge is designed to eliminate those silos by providing a very flexible data collections and distribution mechanism all using the same APIs, features and functions.
```

Comment:
Remove extra space before `Today there...` and `This diversity...` and `Ultimately....` and `Fledge is designed...`